### PR TITLE
Smart waiver

### PR DESCRIPTION
**Pull request by sclark**
Prevent users from cheating or skipping the safety video

### CODE REVIEW

**Comment by pkoenig10** [comment]
FYI, better to rebase on top of your develop branch rather than merging.  That way the commit history doesn't get cluttered with merge commits from feature branches.

**Comment by pkoenig10** [comment]
This is a bit of a hack.  I'm fine with this fix for now because you need to get something that works for build week.  But we should track this issue so we can implement a proper fix in the future.

File: public/cheating.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    }
+    h1 { font-size: 100%; color: #f00; line-height: 1.5em; }
+  </style>
+</head>
+
+<body>
+  <!-- This file lives in public/cheating.html -->
+  <div class="dialog">
+    <h1>You tried to do something suspicious</h1>
+    <p>Please go back and try again, this time without trying to cheat the system...we'll be keeping an eye on you!</p>
```
Comment (pkoenig10) [inline public/cheating.html]: Change the error message to indicate why the user is seeing this page (skipping the waiver video).  And the cute error message isn't really necessary, its better for users to use something clear and straightforward.

**Comment by sclark** [comment]
I like the merge commits, but seems like we rebase in this repo, so I will also rebase in the future

**Comment by sclark** [comment]
This may be hacky and should probably be made configurable, but almost anything on the client side can be manipulated, so this was the quickest, most sane way of doing things. Will keep issue #249 open for future consideration.

File: public/cheating.html (hunk 1 above)
Comment (sclark) [inline public/cheating.html]: Will do

**Comment by sclark** [comment]
@pkoenig10 when pulling in updates from another branch, I understand the rebase and merge idea. When merging pull requests, is there a tradition of rebasing and merging in this repository?

**Comment by sclark** [comment]
If not, I prefer to make a merge commit so we can see that the code came from a different feature branch, but if there is a tradition, I wouldn't want to change it.

File: public/cheating.html [hunk 2]
```diff
@@ -24,8 +24,8 @@
 <body>
   <!-- This file lives in public/cheating.html -->
   <div class="dialog">
-    <h1>You tried to do something suspicious</h1>
-    <p>Please go back and try again, this time without trying to cheat the system...we'll be keeping an eye on you!</p>
+    <h1>You tried to do skip the saftey videos</h1>
+    <p>Please watch the entire saftey video before completing the waiver!</p>
```
Comment (timparenti) [inline public/cheating.html]: "Safety" is misspelled on both lines.

Comment (sclark) [inline public/cheating.html]: I'm getting sloppy...Fixing this now.

**Pull request merged**